frontend/src/metabase/collections/components/BaseTableItem.jsx

```diff
@@ -79,7 +79,7 @@ export function BaseTableItem({
       height: 48,
     };
 
-    const icon = { name: item.getIcon().name };
+    const icon = item.getIcon();
```

Comment:
Here's the fix. When getting the official collection icon it will come with `color`, so we shouldn't just get the name from it.